### DISCUSSION TITLE
Welcome to RadeonProRenderUSD Discussions!

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#488] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RogerDass** [comment]
I have built this under Ubuntu 22.04.
I have built to ~/local/RPR and pointed my PYTHONPATH and PXR_PLUGINPATH_NAME to the appropriate locations in ~/local/RPR (also set LD_LIBRARY_PATH to include ~/local/USD/lib (where USD is installed).
When i select RPR from usdview, I just get a black screen and hang. Eventually USDview crashes. Have I done something wrong?

**Reply by RogerDass** [reply]
I get this output from terminal after RPR is selected as renderer in UsdView:

> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'Render Quality' with type 'TfToken' doesn't have a UI implementation...
> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'Render Mode' with type 'TfToken' doesn't have a UI implementation...
> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'OpenColorIO Config Path' with type 'SdfAssetPath' doesn't have a UI implementation...
> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'Cryptomatte Output Path' with type 'SdfAssetPath' doesn't have a UI implementation...
> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'Cryptomatte Output Mode' with type 'TfToken' doesn't have a UI implementation...
> Warning: in GetRendererSettingsList at line 1118 of /home/roger/DEV/USD/pxr/usdImaging/usdImagingGL/engine.cpp -- Setting 'Camera Mode' with type 'TfToken' doesn't have a UI implementation...
> [MSG  @RIF]: Loaded RIF API version: 1.7.3.0x1c7b817e
> 2 warnings generated.
> 13 warnings generated.
> 6 warnings generated.
> 6 warnings generated.
> 11 warnings generated.
> 12 warnings generated.
> 
> 
> Segmentation fault (core dumped)
